common/common.go

```diff
@@ -93,6 +95,8 @@ const (
 	No LogType = 1 << iota
 	// Initialization - output during system initialization
 	Initialization
+	//
+	Info
```

Comment:
We use "Debug" for the same purposes

Reply:
This was done for having separate logs for ngic dp to go in a log file, as previous comments suggested to reuse the logger so, for now i am using the different log level for it. we will find a better way to do it

Reply:
It is confusing - it is library and NGIC is an example. Why one example shouldn't use DEBUG logs to be separated?